src/model/logistic_equation.py

```
@@ -1,96 +1,102 @@
-        
+
         Raises:
             ValueError: パラメータが設定されていない場合
         """
         if self.gamma is None or self.K is None:
             raise ValueError("パラメータ gamma と K を先に設定してください")
-        
-        n_steps = int((t_end - t_start) / dt) + 1
+
+        if dt <= 0:
+            raise ValueError("dt は正である必要があります")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)